### DISCUSSION TITLE
Unable to login to oci with flatpak in linux

### DISCUSSION THREAD

**Discussion by cortopy** [getseabird/seabird#106] [Q&A]
Oracle oci clusters need to run `command: oci` as configured in kubeconfig. However, this doesn't in work in flatpak because it doesn't have access to the host file system. 

Things I've tried:

- Copy `oci` to `~/.local/bin/oci` and change the `command: oci`  in kubeconfig to `command: home/me/.local/bin/oci`  . This doesn't work because `oci` is not a binary, but a python file so all the imports don't work
- Mount the host file system and change `command: oci` to `command: /run/host/usr/bin/oci` but this doesn't work either, because the python imports still fail

So just wondering how people have gone to fix this

**Comment by jgillich** [comment]
`pip install --user oci`
